### ISSUE TITLE
Show package Files and rename Files to Downloads

### ISSUE THREAD

**Issue by techtonik** [pypa/warehouse#946]
https://godoc.org/github.com/gorilla/websocket#pkg-subdirectories

It is very useful to inspect content of a package without downloading it. Both in Debian and in Go. You can also see .dll's and data files that package may install in some doubtful locations. I propose to steal the concept from Go - look for Files and Directories links (although their meaning is a little different).

**Comment by nlhkabu** [comment]
'Files' has been renamed to 'Download Files' (to differentiate from 'Download Statistics').

**Comment by techtonik** [comment]
I'd shorten this to just `Downloads` and `Statistics`. Although it is not related to this issue.

**Comment by nlhkabu** [comment]
I don't disagree with you on that.  If you want to make a PR with that change, I'd be happy to merge it.

**Comment by techtonik** [comment]
@nlhkabu I extended the idea a little - ttps://github.com/pypa/warehouse/pull/974
I don't have local copy, so I don't know how it looks/

**Comment by di** [comment]
@techtonik See https://warehouse.readthedocs.org/development/getting-started/ if you need information about setting Warehouse up locally.

**Comment by brainwane** [comment]
Given that we have renamed "Files" to "Download Files", I've renamed the ticket for clarity. (Also, we have removed the statistics pane, and have an open TODO (#787) to implement statistics and integrate stats back into the UI.)

Nicole is running some user tests in the near future and that data will help her see what users look for when they're trying to find more information about a package, so we can decide whether and how to prioritize this feature. Thanks!

**Comment by miketheman** [comment]
@di is Inspector intended to support this use case?

**Comment by di** [comment]
I think there's a big 'TBD' on how Inspector should be used with PyPI, which we should probably figure out separately. At the moment though, ita not really designed to be integrated with PyPI like this.

**Comment by nlhkabu** [comment]
Note that npm have a "code" tab which displays the package contents - see [this blog post](https://github.blog/changelog/2022-12-06-explore-the-content-of-your-npm-packages-with-the-new-code-explorer/) for more info.

<img width="1003" height="793" alt="Image" src="https://github.com/user-attachments/assets/68c63597-6da8-496d-90ae-beadd5472557" />

I recently participated in some user research looking at how users assess the security and trustworthiness of the packages they install ([project context](https://github.com/ossf/tac/issues/424)). Our testing highlighted that this functionality was valued by users as part of their overall security assessment process.  

Note that we would need to test how to display this in the UI, as "code" is easily confused with "source code".

@di can you share a bit more about Inspector here?

**Comment by di** [comment]
> @di can you share a bit more about Inspector here?

Not much has changed since my [previous comment](https://github.com/pypi/warehouse/issues/946#issuecomment-1257288129) three years ago. I think we'd still want Inspector to be a separate service from PyPI (to mitigate handling/unpacking arbitrary untrusted user-submitted code) but there's probably a path towards providing a similar "code" tab.

I think this would depend on https://github.com/pypi/inspector/issues/27 as well as some other performance improvements for Inspector happening first, and then some interface that PyPI can use it by (not exactly sure what this would be).
